scripts/version.sh

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+shopt -s nullglob
+
+root=$1
+
+[[ "$(toml get --raw Cargo.toml package.version)" =~ ([0-9]+)\.([0-9]+)\.([0-9]+) ]]
+splitVersion=( "${BASH_REMATCH[@]:1}" )
+
+majorChanges=( "$root"/changes/unreleased/major/*.md )
+mediumChanges=( "$root"/changes/unreleased/medium/*.md )
+minorChanges=( "$root"/changes/unreleased/minor/*.md )
+
+if (( ${#majorChanges[@]} > 0 )); then
+  (( splitVersion[0]++ )) || true
```

Comment:
why suppress errors with `|| true`?

Reply:
Added an explanation now: https://github.com/NixOS/nixpkgs-check-by-name/blob/b1166b31a9d70672c1b55c1eb04dbbacbbc83818/scripts/version.sh#L16-L18

Reply:
lol that explains an issue I couldn't figure out a couple of weeks ago where I was using ++ and had to change it to the `v = v + 1` form to get it to work.